herring/puzzles/migrations/0017_userprofile_subscriptions.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.16 on 2024-01-09 05:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('puzzles', '0016_auto_20240109_0509'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
What is the expected format of the field? It looks like either space-separated or comma-separated might work ok?